### DISCUSSION TITLE
[Config Support]: wifi camera configuration best practices

### DISCUSSION THREAD

**Discussion by turboc1208** [blakeblackshear/frigate#10159] [Config Support]
### Describe the problem you are having

Two out of nine of my reolink cameras are wifi.  I have a tool shed on the back of my property.  The nearest point I can run a cable from is far enough away to make running a cable expensive.  But my wifi reaches pretty well.  I also have a reolink wifi doorbell which works perfect.  It's the two on the shed that are the problem.  What I'm looking for are best practices in the config for setting up wireless cameras.  I understand that in a perfect world all my cameras would be wired.  Unfortunately my world is not perfect.  

### Version

0.13.2-6476F8A

### Frigate config file

```yaml
Thats kind of what I am asking for.
```

### Relevant log output

```shell
NA
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Any other information that may be helpful

Looking for general discussion, not necessarily solutions.

**Comment by supermarkert** [comment]
I don't believe there's anything special to do for Wifi cameras vs Ethernet cameras.  
For my Reolink Wifi doorbell, I just followed [the recommended RTSP settings](https://docs.frigate.video/configuration/camera_specific) and it works fine without errors.  
If you don't have a good Wifi connection, you're going to get errors. I don't think there's anything you can do in Config to avoid them.

**Comment by 18923** [comment]
I have a similar situation. If you have power at the shed try a "powerline extender". I connected a router in bridge mode to the extender in the garage about 150 feet (45M) away. My cameras are still WIFI connected but the signal is much stronger and working 99.99% 
Occasionally the connection freezes and I have to reboot the extenders to recover.
